apps/minifront/src/state/ibc-out.ts

```diff
@@ -215,20 +216,44 @@ const getPlanRequest = async ({
 
   const { timeoutHeight, timeoutTime } = await getTimeout(chain.channelId);
 
+  // Request transparent address from view service
+  const { address: t_addr, encoding: _encoding } = await penumbra
+    .service(ViewService)
+    .transparentAddress(new TransparentAddressRequest({}));
+  if (!t_addr) {
+    throw new Error('Error with generating IBC transparent address');
+  }
+
+  // IBC-related fields
+  let denom = getMetadata(selection.balanceView).base;
+  let useTransparentAddress = false;
+
+  // Temporary: detect USDC Noble withdrawals, and use transparent (t-addr) addresses
+  // to ensure bech32m encoding compatibility.
+
+  if (denom.includes('uusdc') && chain.chainId == 'noble-1') {
+    // Use the t-addr for USDC withdrawals, and don't override existing
+    // 'useCompatAddress' field to maintain backwards compatibility.
+    useTransparentAddress = true;
+    // Set the return address to the t-addr.
+    returnAddress = t_addr;
+  }
+
   return new TransactionPlannerRequest({
     ics20Withdrawals: [
       {
         amount: toBaseUnit(
           BigNumber(amount),
           getDisplayDenomExponentFromValueView(selection.balanceView),
         ),
-        denom: { denom: getMetadata(selection.balanceView).base },
+        denom: { denom },
         destinationChainAddress,
         returnAddress,
         timeoutHeight,
         timeoutTime,
         sourceChannel: chain.channelId,
         useCompatAddress: bech32ChainIds.includes(chain.chainId),
+        useTransparentAddress,
```

Comment:
on this branch, performing a regular IBC withdrawal with `useTransparentAddress` set to false is throwing a binding signature failure. cc @erwanor  

<img width="1728" alt="Screenshot 2024-12-18 at 11 53 30 AM" src="https://github.com/user-attachments/assets/a6b386d7-cd89-44b4-8f75-da808175b471" />

Reply:
What happened here?

Reply:
this was pre-upgrade, resolved now

Reply:
I think this happened using a new client on an old network, with changes to the proto actions, the auth hash looks invalid to the old full node (because the extra proto fields are ignored when creating the domain type).